tests/test_views.py

```diff
@@ -196,7 +196,7 @@ def test_create_and_fetch(bukudb, monkeypatch, client, fetch, title, desc):
     ('_continue_editing', '/bookmark/edit/', {'id': '1', 'url': '/bookmark/'}),
 ])
 def test_create_redirect(client, redirect, uri, args):
-    query = {'link': 'http://example.com', 'title': '', 'description': '', 'tags': '', redirect: 'on'}
+    query = {'link': 'http://example.com', 'title': '', 'description': '', 'tags': '', 'fetch': '', redirect: 'on'}
```

Comment:
There's no need to do a remote fetch for this test

(This change improves performance of this test depending on network state)